vendor/TraceKit/tracekit.js

```diff
@@ -35,6 +34,13 @@ TraceKit.wrap = function traceKitWrapper(func) {
     return wrapped;
 };
 
+TraceKit.getLocationHref = function () {
+    if (typeof document === 'undefined')
+        return '';
+
+    return document.location.href;
```

Comment:
Could we maybe use `window.location.href` instead? And avoid the `document` issues?